in_text/text_data_release.yml

```
@@ -476,34 +482,34 @@ entities:
     -
       attr-label: TE
       attr-def: >-
-        XX
+        transfer entropy between source and sink
       attr-defs: >-
         This release
       data-min: -3.00461E-16
       data-max: 0.50502906
-      data-units: XX
+      data-units: unitless since transfer entropy is of nats then normalized by the entropy of the sink variable
     -
       attr-label: TEcrit
       attr-def: >-
-        XX
+        the significance threshold for transfer entropy
```

How is this defined?
```suggestion
        The significance threshold for transfer entropy
```